### ISSUE TITLE
Error when deploying to Heroku

### ISSUE THREAD

**Issue by bzigterman** [themarshallproject/klaxon#202]
I’m trying to set up Klaxon for our newsroom, and I ran into this error when I hit the “Deploy App” button on Heroku’s website and it was in the “Build App” section:

```
-----> Ruby app detected
-----> Compiling Ruby/Rails
       Command: 'set -o pipefail; curl -L --fail --retry 5 --retry-delay 1 --connect-timeout 3 --max-time 30 https://s3-external-1.amazonaws.com/heroku-buildpack-ruby/heroku-18/ruby-2.3.4.tgz -s -o - | tar zxf - ' failed on attempt 1 of 3.
       Command: 'set -o pipefail; curl -L --fail --retry 5 --retry-delay 1 --connect-timeout 3 --max-time 30 https://s3-external-1.amazonaws.com/heroku-buildpack-ruby/heroku-18/ruby-2.3.4.tgz -s -o - | tar zxf - ' failed on attempt 2 of 3.
 !
 !     An error occurred while installing ruby-2.3.4
 !     
 !     This version of Ruby is not available on Heroku-18. The minimum supported version
 !     of Ruby on the Heroku-18 stack can found at:
 !     
 !     https://devcenter.heroku.com/articles/ruby-support#supported-runtimes
 !
 !     Push rejected, failed to compile Ruby app.
 !     Push failed

```

Any idea why this is happening? I’m not a developer, so I don’t really know what to do next. Sorry if this is the wrong place to post this. And thanks!

**Comment by tommeagher** [comment]
Hi @bzigterman, it looks like the brand-new Heroku stacks don't support Ruby 2.3: https://devcenter.heroku.com/changelog-items/1485
We'll have to upgrade Klaxon to address this. Thanks for flagging it.

**Comment by GabeIsman** [comment]
@bzigterman This should be fixed now with the 0.3.0 release. Give it another shot!
